### DISCUSSION TITLE
Stitching tiled images

### DISCUSSION THREAD

**Discussion by akandrew332** [angelolab/toffy#275] [Q&A] [answered]
I've acquired a bunch of tiled acquisitions.  Is there a script that will stitch these together?

**Comment by ngreenwald** [comment, marked as answer]
Yes, that code is in `ark` since we usually do it after we've segmented and clustered the cells, but if you want to visualize the channel data before then it's here: https://github.com/angelolab/ark-analysis/blob/main/templates/Image_Stitching.ipynb

**Reply by akandrew332** [reply]
Thanks so much!
